tests/apps/core/test_core_factories_labels.py

```diff
@@ -0,0 +1,18 @@
+"""
+Unit tests for the Label factory
+"""
+from django.test import TestCase
+
+from magnify.apps.core.factories import LabelFactory
+
+
+class LabelsFactoriesTestCase(TestCase):
+    """
+    Unit test suite to validate the behavior of the Label factory
+    """
+
+    def test_factories_labels_color(self):
+        """The color field should be populated with a valid color."""
+        label = LabelFactory()
+        self.assertTrue(label.color.startswith("#"))
+        self.assertTrue(len(label.color), 7)
```

Comment:
I don't know how to write this in an assert but the regex accepts colors like #111

Reply:
Yes but our factory only generates colors with 6 digits if I judge by this: https://github.com/joke2k/faker/blob/master/faker/providers/color/fr_FR/__init__.py

Any way we are not trying to test what the model accepts but that the factory generates valid random colors, so it should be good enough?